sdk/servicebus/service-bus/test/internal/sendBatch.spec.ts

```diff
@@ -471,4 +472,42 @@ describe("Send Batch", () => {
     await sender.sendMessages(batch);
     await serviceBusClient.test.verifyAndDeleteAllSentMessages(entityNames, messagesToSend);
   });
+
+  it("send() with null/undefined properties", async () => {
+    await beforeEachTest(TestClientType.UnpartitionedQueue);
+
+    const messageId = `null/undefined properties: ${Date.now()}`;
+
+    await sender.sendMessages({
+      messageId,
+      body: undefined,
+      applicationProperties: {
+        nullProperty: null,
+        // the type definition hasn't opened up to allow undefined (we do allow)
+        // null, however.
+        undefinedProperty: undefined as any,
```

Comment:
Instead of treating values of `undefined`, as `null`, should we instead treat it as though it doesn't exist?

Reply:
This is rhea, as you said. The type definition doesn't show undefined, so people should be aware of it if they're using TS and if they're using JS at least we document that the only valid return values are still just 'null'.